pkgs/applications/networking/newsreaders/raven-reader/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "raven-reader";
+  version = "1.0.72";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/hello-efficiency-inc/${pname}/releases/download/v${version}/Raven-Reader-${version}.AppImage";
```

Comment:
```suggestion
    url = "https://github.com/hello-efficiency-inc/raven-reader/releases/download/v${version}/Raven-Reader-${version}.AppImage";
```

Reply:
Could you elaborate on why this is better? Isn't it better (less redundancy) or at least as good to substitute with the already defined `pname`?